markers/sauce.py

```diff
@@ -18,3 +19,11 @@ def pytest_configure(config):
             config.option.markexpr = 'sauce and ({})'.format(config.option.markexpr)
         else:
             config.option.markexpr = 'sauce'
+
+
+def pytest_runtest_setup(item):
+    browser.ensure_browser_open(item)
+
+
+def pytest_runtest_teardown():
```

Comment:
nextitem isn't part of the usual test protocol calls

nextitem is only available with the runtestprotocol hook